articles/storage/blobs/storage-blob-performance-tiers.md

```diff
@@ -51,6 +51,13 @@ Premium performance storage is ideal for workloads that require fast and consist
 
 Standard performance supports different [access tiers](storage-blob-storage-tiers.md) to store data in the most cost-effective manner. It's optimized for high capacity and high throughput on large data sets.
 
+-**Backup and disaster recovery datasets**. Standard performance storage offers cost-efficient tiers making it a perfect use case for both short/long-term disaster recovery datasets, secondary backups, and compliance data archiving.
```

Comment:
Please use actual words like "and" and "or" instead of slashes (i.e. "short/long-term" should be "short and long-term").